.sonarcloud.properties

```diff
@@ -0,0 +1,2 @@
+# Disable cognitive complexity in files
+sonar.exclusions=library/src/components/Schema.tsx
```

Comment:
I think it excludes all not just complexity, please ignore complexity on a file-level maybe -> https://github.com/asyncapi/parser-js/blob/master/test/testsUtils.js#L23

Reply:
I tried to use the `eslint` and it doesn't work. I know about that sonar exclude all rules, but I cannot find example how to exclude only particular rule, I also cannot find the title/id of the complexity rule...

Also related to the package-lock.json - lerna's `bootstrap` removes the redundant modules in the monorepo :) I will revert this.